src/main/java/com/jcabi/github/Status.java

```diff
@@ -0,0 +1,22 @@
+package com.jcabi.github;
+
+import com.jcabi.aspects.Immutable;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+
+@Immutable
+public interface Status {
+
+    @NotNull(message = "state is never NULL")
+    StatusState state();
+
+    // TODO optional
+    String targetUrl();
```

Comment:
@zygm0nt If some functionality is not implemented we expect a proper todo is created
See the article for inspiration http://www.yegor256.com/2009/03/04/pdd.html